wrappers/javascript/aries-askar-react-native/aries-askar.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "12.0" }
+  s.platforms    = { :ios => "14.0" }
```

Comment:
The comment in the other file mentions 13 and up. What will this support? Could we easily support lower versions? 14 is still quite new-ish (and higher than minimum bifold version)

Reply:
I could not find any metrix that links rust to iOS version. I will try 1.60.x - 1,68.2 and see which has the lowest version compatibility.

Reply:
Hmmm, I tried to look for a lower build version and all seem to target 16.2. I think I misread the 13 from somewhere...

They added dylib support in rust for iOS, back, at 1.65.0. I checked all versions above and they all have a `minos` version of 16.2 which is 67% of the iPhones (https://iosref.com/ios-usage). 

If we must have support for lower versions we need to move back to static libraries.

Reply:
16.2 is not acceptable

Reply:
I also tested this without askar and just a `Hello World` example and they all also resolve to `16.2`.

Reply:
> 16.2 is not acceptable

Okay. Back to static libraries it is.

Reply:
This seems a bit weird to me, as one of my test handsets is an iPhone SE (1st generation) running iOS 15.7.3 and it somehow has been working so far with all askar package dev releases up to last week or so. What could be the reason for that?

Reply:
Well, it looks like this issue does not affect any functionality in development. I'm also was able to run the app on iOS 14 in development, using Metro bundler.

And it's also possible  to create a iOS app bundle, but the problem is that it will fail bundle validation required to distribute the app (for example, upload the app to Apple Testflight). 

For details, please see https://github.com/hyperledger/aries-askar/issues/135.